assets/js/components/wp-dashboard/common.stories-GA4.js

```diff
@@ -0,0 +1,165 @@
+/**
+ * Shared exports among WPDashboard GA4 Stories.
```

Comment:
I think it's better to rename it to `common-GA4.stories.js` to keep the stories extension as `*.stories.js`

WDYT @techanvil?